quinn-proto/src/connection/mod.rs

```diff
@@ -1309,6 +1309,20 @@ impl Connection {
         self.path.congestion.as_ref()
     }
 
+    /// Resets path-specific settings.
+    ///
+    /// This will force-reset several subsystems related to a specific network path.
+    /// Currently this is the congestion controller, round-trip estimator, and the MTU
+    /// discovery.
+    ///
+    /// This is useful when it is known the underlying network path has changed and the old
+    /// state of these subsystems is no longer valid or optimal. In which case it might be
```

Comment:
```suggestion
    /// state of these subsystems is no longer valid or optimal, in which case it might be
```

Reply:
This creates a very long sentence, I'm used to technical writing style guides recommending shorter sentences.  I've reworded this slightly differently, hopefully also improving this sentence, but feel free to ask for more tweaks.